### DISCUSSION TITLE
How is perf tested?

### DISCUSSION THREAD

**Discussion by merisbahti** [reduxjs/react-redux#1692] [Q&A] [answered]
In this talk: https://www.youtube.com/watch?v=yOZ4Ml9LlWE

@markerikson  mentions how they noticed that perf went down.

I'm curious where this test suite is located?

Or if it isn't a suite, how did you measure perf?

I've tried looking around in the `reduxjs` repos but I couldn't find anything.

Thanks, and have a good weekend.

**Comment by markerikson** [comment, marked as answer]
We used this repo here: https://github.com/reduxjs/react-redux-benchmarks
